notebooks/04_model_select_and_optimize/azureml_hyperdrive_tuning_svd.ipynb

```diff
@@ -0,0 +1,731 @@
+       "  'runTemplate': 'HyperDrive',\n",
+       "  'azureml.runsource': 'hyperdrive'},\n",
+       " 'logFiles': {'azureml-logs/hyperdrive.txt': 'https://anargyri6699545766.blob.core.windows.net/azureml/ExperimentRun/dcid.movielens_100k_svd_model_1550672871383/azureml-logs/hyperdrive.txt?sv=2018-03-28&sr=b&sig=dt%2BtkHqz49%2F2sQTmTzHDMh%2BchdRdiMSELDnQycGsE3g%3D&st=2019-02-20T14%3A50%3A26Z&se=2019-02-20T23%3A00%3A26Z&sp=r'}}"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create an experiment to track the runs in the workspace\n",
+    "exp = aml.core.Experiment(workspace=ws, name=EXP_NAME)\n",
+    "run = exp.submit(config=hd_config)\n",
+    "\n",
+    "azureml.widgets.RunDetails(run).show()\n",
+    "run.wait_for_completion(show_output=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get best run and printout metrics\n",
+    "best_run = run.get_best_run_by_primary_metric()\n",
+    "\n",
+    "best_run_metrics = best_run.get_metrics()\n",
+    "parameter_values = best_run.get_details()['runDefinition']['Arguments']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'Number of epochs': 30,\n",
+       " 'rmse': 1.1324249983589592,\n",
+       " 'precision_at_k': 0.17932131495228,\n",
+       " 'ndcg_at_k': 0.20470147010861087}"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "best_run_metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--datastore $AZUREML_DATAREFERENCE_workspaceblobstore --train-datapath data/movielens_100k_train.pkl --test-datapath data/movielens_100k_test.pkl --surprise-reader ml-100k --rating-metrics rmse --ranking-metrics precision_at_k ndcg_at_k --usercol userID --itemcol itemID --ratingcol rating --k 10 --random-state 0 --epochs 30 --biased --verbose --n_factors 10 --init_mean -0.320173377588374 --init_std_dev 0.10636418973087 --lr_bu 1E-06 --lr_bi 1E-06 --lr_pu 0.0171152526860302 --lr_qi 0.1 --reg_bu 0.180199641033639 --reg_bi 0.963286774529248 --reg_pu 0.994667973995922 --reg_qi 0.673467023866025\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\" \".join(parameter_values))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    shutil.rmtree(SCRIPT_DIR)\n",
+    "    shutil.rmtree(DATA_DIR)\n",
+    "except (PermissionError, FileNotFoundError):\n",
+    "    pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
I think it would be good to have some concluding remarks like in the W&D hyperdrive notebook

Reply:
Agree.